docs/source/index.rst

```diff
@@ -40,6 +40,26 @@ Produces
 .. runblock:: console
 
     $ date 
+    
+Should a user desire to omit some lines:
+
+.. code-block:: rst
+
+    .. runblock:: pycon
+        
+        >>> setup_function(args) #ignore
```

Comment:
This should be `  # ignore` (note the two spaces before and single space after the hash. Otherwise, that code won't be PEP-8-complaint and will trigger linters.